### PR TITLE
allow use static attributes from group in entityNameExp of group

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,2 +1,3 @@
+- Add: allow use static attributes from group in entityNameExp of group
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: Being a new thing ("Add"...), would it need modifications to documentation to describe the new capability? Or documentation is already describing this?

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE]: Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1357/commits/8f5489459b775dd5b272f65f61bac179765df5ed
It is a fix, static attrs are expected to be in context of expression evaluation in all cases.
Maybe a doc should include it

**Review by mapedraza — APPROVED** [review]
LGTM

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (mapedraza) [reply CHANGES_NEXT_RELEASE]: This also applies for the other places where expressions are evaluated? (measure transformation, commands and endpoint)?

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE]: Doc updated in: https://github.com/telefonicaid/iotagent-node-lib/pull/1357/commits/82956a7bf557dbdf77d1068ec67d29be102d1f16

Comment (AlvaroVega) [reply CHANGES_NEXT_RELEASE]: Still measures are not avaiable for entityNameExp.
Measures, statics, attributes are avaiable in attribute expressions and attribute entity_name context expressions

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**